common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -158,6 +159,7 @@ object WhiskConfig {
     val mainDockerEndpoint = "main.docker.endpoint"
     val routerDockerEndpoint = "router.docker.endpoint"
     val elkDockerEndpoint = "elk.docker.endpoint"
+    val catalogDockerEndpoint = "catalog.docker.endpoint"
```

Comment:
this will look like dead code in openwhisk since it's not used in openwhisk.
is there a blue whiskconfig that is a better home for this?